src/components/ProductLinkGrid/ProductLinkGrid.tsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @next/next/no-img-element */
+import { ProductLink, productLinkPropsAdapter } from "@/components/ProductLink";
+import { ProductType } from "@/lib/shopify";
+import styles from "./styles.module.scss";
+
+export interface ProductLinkGridProps {
+  products: ProductType[];
+}
+
+export const ProductLinkGrid: React.FC<ProductLinkGridProps> = ({
+  products,
+}) => {
+  return (
+    <div>
```

Comment:
@sprajams added a LayoutGrid component recently that could be used in place of this div. Adding it in would be similar to how CollectionLinkGrid does but make sure this components span the correct amount of columns.